society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -656,42 +828,118 @@
       <text variable="URL"/>
     </group>
   </macro>
-  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true">
-    <layout suffix="." delimiter="; ">
-      <choose>
-        <if position="ibid-with-locator">
-          <group delimiter=", ">
-            <text term="ibid"/>
-            <text macro="point-locators-subsequent"/>
-          </group>
-        </if>
-        <else-if position="ibid">
-          <text term="ibid"/>
-        </else-if>
-        <else-if position="subsequent">
+  <macro name="unsigned-dictionary-note">
+    <group delimiter=", ">
+      <group>
+        <text variable="container-title" font-style="italic" text-case="uppercase"/>
```

Comment:
really uppercase? What's an example here?

Reply:
The only thing  that gets to this macro should be an unsigned dictionary article. The formatting is from https://sblhs2.com/2017/03/30/citing-reference-works-2-lexica/.

I am fine removing the uppercase and just entering dictionary titles as uppercase in Zotero if that is a better way to handle it. Let me know and I can change it and submit a new commit.